conformance/conformance_python.py

```diff
@@ -53,18 +54,25 @@ class ProtocolError(Exception):
   pass
 
 def do_test(request):
-  test_message = test_messages_proto3_pb2.TestAllTypes()
+  isProto3 = (request.message_type == "protobuf_test_messages.proto3.TestAllTypes")
```

Comment:
It seems that only python_cpp can use [generated descriptor_pool ](https://github.com/google/protobuf/blob/master/python/google/protobuf/descriptor_pool.py#L990).